content/en/dashboards/sharing.md

```diff
@@ -122,12 +122,12 @@ Datadog has a [dedicated API][7] allowing you to interact with your shared graph
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: https://app.datadoghq.com/dashboard/lists
+[1]: /dashboard/lists
```

Comment:
made the links relative, I assume this will make them work on other sites (EU, US3, US5)

Reply:
oh hmm I guess that only works for doc links. Will revert that part of the change